content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Traces For Kubernetes System Components
+reviewers:
+- logicalhan
+- lilic
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
```

Comment:
This should use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) to show that it's not a generally available feature.

Reply:
done